flake.nix

```diff
@@ -78,69 +62,65 @@
-          ./modules/eth/lighthouse-beacon.nix
-          ./modules/eth/mev-boost.nix
-          ./system/global.nix
-          ./system/ramdisk.nix
-          ./home-manager/core.nix
-          home-manager.nixosModules.home-manager
-          disko.nixosModules.disko
-          {
-            nixpkgs.overlays = [
-              ethereum-nix.overlays.default
-              outputs.overlays.additions
-              outputs.overlays.modifications
-            ];
-          }
-          {
-            home-manager.sharedModules = [
-              sops-nix.homeManagerModules.sops
-            ];
-          }
-        ];
-        customFormats = customFormats;
-        format = "kexecTree";
-      };
-
-      "dinar-ephemeral-alpha" = nixos-generators.nixosGenerate {
-        system = "x86_64-linux";
-        specialArgs = { inherit inputs outputs; };
-        modules = [
-          ./hosts/dinar-ephemeral-alpha
-          ./modules/eth/erigon.nix
-          ./modules/eth/lighthouse-beacon.nix
-          ./modules/eth/mev-boost.nix
-          ./system/global.nix
-          ./home-manager/core.nix
-          home-manager.nixosModules.home-manager
-          disko.nixosModules.disko
-          {
-            nixpkgs.overlays = [
-              ethereum-nix.overlays.default
-              outputs.overlays.additions
-              outputs.overlays.modifications
-            ];
-          }
-          {
-            home-manager.sharedModules = [
-              sops-nix.homeManagerModules.sops
-            ];
-          }
-          {
-            # GRUB timeout
-            boot.loader.timeout = nixpkgs.lib.mkForce 1;
+      nixosConfigurations = {
+        # nix build .#nixosConfigurations.ponkila-ephemeral-beta.config.system.build.kexecTree
+        ponkila-ephemeral-beta = nixpkgs.lib.nixosSystem {
+          system = "x86_64-linux";
+          specialArgs = { inherit inputs outputs; };
+          modules = [
+            ./hosts/ponkila-ephemeral-beta
+            ./modules/eth/erigon.nix
+            ./modules/eth/lighthouse-beacon.nix
+            ./modules/eth/mev-boost.nix
+            ./system/formats/netboot-kexec.nix
+            ./system/global.nix
+            ./system/ramdisk.nix
+            ./home-manager/core.nix
+            home-manager.nixosModules.home-manager
+            disko.nixosModules.disko
+            {
+              nixpkgs.overlays = [
+                ethereum-nix.overlays.default
+                outputs.overlays.additions
+                outputs.overlays.modifications
+              ];
+            }
+            {
+              home-manager.sharedModules = [
+                sops-nix.homeManagerModules.sops
+              ];
+            }
+          ];
+        };
 
-            # Load into a tmpfs during stage-1
-            boot.kernelParams = [ "copytoram" ];
-          }
-        ];
-        format = "install-iso";
+        # nix build .#nixosConfigurations.dinar-ephemeral-alpha.config.system.build.isoImage
```

Comment:
We could also alias this in a separate flake attribute, .e.g:

`dinar-ephemeral-alpha = nixosConfigurations.dinar-ephemeral-alpha.config.system.build.isoImage` and so. Technically, these declarations should be under `packages`, but it might be a bad idea since it implies cross-compilation in CI.